<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>AIDA Lab</title>
        <!-- <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" /> -->
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">AIDA Lab</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/aida_lab.jpg" alt="" /></span>
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#page-top"><strong>AIDA Lab</strong></a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#people">People</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#projects">Projects</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Awards</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#theses">Master Theses</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        <span class="text-primary">AIDA Lab</span>
                    </h1>
                    <h2 class="mb-0">
                        <span class="text-primary">Algorithms for Intelligent Data Analytics</span>
                    </h2>
                    <br>
                    <p class="lead mb-5">	We are a research group in the <a href="https://www.dei.unipd.it/en/">Department of
Information Engineering</a> of <a href="https://www.unipd.it/en/">University of Padova</a>.</p>

                    <p class="lead mb-5">
                    Our research is focused on the development of <strong>algorithms for extracting useful information from large datasets</strong>. Our work spans areas such as <strong>theory of computation</strong>, <strong>data mining</strong>, <strong>machine learning</strong> with applications to the analysis of big data from <strong>social networks, biomedical data, urban traffic, industrial processes</strong>.</p>

                    <p class="lead mb-5">
                      We focus on <strong>algorithms for: modern computing architectures; pattern mining and learning; temporal, dynamic, and evolving data; unsupervised learning (clustering, diversity)</strong>.</p>

                  <p class="lead mb-5">
                    The common themes of our work include <em>scalability</em>, <em>fairness and privacy</em>, <em>networked data</em>, and <em>rigorous guarantees</em>.<p>
                    <!--
                    <div class="social-icons">
                        <a class="social-icon" href="#!"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="#!"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="#!"><i class="fab fa-twitter"></i></a>
                        <a class="social-icon" href="#!"><i class="fab fa-facebook-f"></i></a>
                    </div>
                  -->
                </div>
            </section>
            <hr class="m-0" />
            <!-- People-->
            <section class="resume-section" id="people">
                <div class="resume-section-content">
                    <h2 class="mb-5">People</h2>
                    <h3 class="mb-0">Faculty</h3>
                    <ul>
                      <li><a href="https://www.dei.unipd.it/~ceccarello/">Matteo Ceccarello</a> (Assistant Professor)</li>
                      <li><a href="https://www.dei.unipd.it/~pellegri/">Leonardo Pellegrina</a> (Assistant Professor)</li>
                      <li><a href="https://www.dei.unipd.it/~capri/">Andrea Alberto Pietracaprina</a> (Full Professor)</li>
                      <li><a href="https://www.dei.unipd.it/~geppo/">Geppino Pucci</a> (Full Professor)</li>
                      <li><a href="https://www.dei.unipd.it/~silvestri/">Francesco Silvestri</a> (Associate Professor)</li>
                      <li><a href="https://www.dei.unipd.it/~vandinfa/">Fabio Vandin</a> (Full Professor)</li>
                    </ul>
                    <h3 class="mb-0">Postdocs and PhD students</h3>
                    <ul>
                      <li>Cristian Boldrin (Ph.D student)</li>
                      <li>Fabrizio Boninsegna (Phd student)</li>
                      <li>Antonio Collesei (Phd student)</li>
                      <li>Dario Simionato (PhD student)</li>
                      <li>Giorgio Venturin (Ph.D student)</li>
                    </ul>
                    <div class="subheading mb-3">Alumni</div>
                    <ul>
                      <li>Andrea Tonon (now @ Huawei)</li>
                      <li><a href="https://diegosantoro.github.io/">Diego Santoro</a></li>
                      <li><a href="https://iliesarpe.github.io/">Ilie Sarpe</a> (now @ KTH)</li>
                    <ul>

            </section>
            <hr class="m-0" />
            <!-- Publications-->
            <section class="resume-section" id="publications">
                <div class="resume-section-content">
                    <h2 class="mb-5">Recent Publications</h2>
<!-- PUBS -->
<h3 class="mb-0">2023</h3><ul><li>Andrea Tonon, Fabio Vandin <emph><a href="https://doi.org/10.1007/s10115-022-01800-7">caSPiTa: mining statistically significant paths in time series data from an unknown network.</a></emph> Knowl. Inf. Syst.</li>
<li>Dario Simionato, Fabio Vandin <emph><a href="https://doi.org/10.24963/ijcai.2023/726">Bounding the Family-Wise Error Rate in Local Causal Discovery Using Rademacher Averages (Extended Abstract).</a></emph> IJCAI</li>
<li>Enrico Dandolo, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.1007/978-3-031-39698-4_32">Distributed k-Means with Outliers in General Metrics.</a></emph> Euro-Par</li>
<li>Leonardo Pellegrina <emph><a href="https://doi.org/10.1145/3580305.3599325">Efficient Centrality Maximization with Rademacher Averages.</a></emph> KDD</li>
<li>Matteo Ceccarello, Andrea Pietracaprina, Geppino Pucci, Federico Soldà <emph><a href="https://doi.org/10.1186/s40537-023-00717-4">Scalable and space-efficient Robust Matroid Center algorithms.</a></emph> J. Big Data</li>
<li>Matteo Ceccarello, Anton Dignös, Johann Gamper, Christina Khnaisser <emph><a href="https://doi.org/10.1145/3603719.3603732">Indexing Temporal Relations for Range-Duration Queries.</a></emph> SSDBM</li>
<li>Paolo Pellizzoni, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.1007/978-3-031-38906-1_41">Fully Dynamic Clustering and Diversity Maximization in Doubling Metrics.</a></emph> WADS</li>
<li>Paolo Pellizzoni, Fabio Vandin <emph><a href="https://doi.org/10.1109/ICDE55515.2023.00190">VC-dimension and Rademacher Averages of Subgraphs, with Applications to Graph Mining.</a></emph> ICDE</li>
<li>Shiyuan Deng, Francesco Silvestri , Yufei Tao  <emph><a href="https://doi.org/10.4230/LIPIcs.ICDT.2023.4">Enumerating Subgraphs of Constant Sizes in External Memory.</a></emph> ICDT</li></ul>
<h3 class="mb-0">2022</h3><ul><li>A. Guiotto, G. Bortolami, A. Ciniglio, F. Spolaor, G. Guarneri, A. Avogaro, F. Cibin, F. Silvestri, Z. Sawacha. <emph><a href="https://www.sciencedirect.com/science/article/abs/pii/S0966636222005537">Machine learning approach to diabetic foot risk classification with biomechanics data</a></emph> Gait & Posture</li>
<li>Adam Charane, Matteo Ceccarello, Anton Dignös, Johann Gamper <emph><a href="https://doi.org/10.1007/978-3-031-09850-5_17">Efficient Computation of All-Window Length Correlations.</a></emph> DB&amp;IS</li>
<li>Andrea Tonon, Fabio Vandin <emph><a href="https://doi.org/10.1007/s10115-022-01689-2">gRosSo: mining statistically robust patterns from a sequence of datasets.</a></emph> Knowl. Inf. Syst.</li>
<li>Dario Simionato, Fabio Vandin <emph><a href="https://doi.org/10.1007/978-3-031-26419-1_16">Bounding the Family-Wise Error Rate in Local Causal Discovery Using Rademacher Averages.</a></emph> ECML/PKDD</li>
<li>Davide Buffelli, Fabio Vandin <emph><a href="https://doi.org/10.3390/data7010010">The Impact of Global Structural Information in Graph Neural Networks Applications.</a></emph> Data</li>
<li>Davide Buffelli, Fabio Vandin <emph><a href="https://doi.org/10.1109/IJCNN55064.2022.9892010">Graph Representation Learning for Multi-Task Settings: a Meta-Learning Approach.</a></emph> IJCNN</li>
<li>Davide Buffelli, Pietro Lió, Fabio Vandin <emph><a href="https://papers.nips.cc/paper_files/paper/2022/hash/ceeb3fa5be458f08fbb12a5bb783aac8-Abstract-Conference.html">SizeShiftReg: a Regularization Method for Improving Size-Generalization in Graph Neural Networks</a></emph> NeurIPS</li>
<li>Davide Buffelli, Pietro Lió, Fabio Vandin <emph><a href="http://papers.nips.cc/paper_files/paper/2022/hash/ceeb3fa5be458f08fbb12a5bb783aac8-Abstract-Conference.html">SizeShiftReg: a Regularization Method for Improving Size-Generalization in Graph Neural Networks.</a></emph> NeurIPS</li>
<li>Diego Santoro, Ilie Sarpe <emph><a href="https://doi.org/10.1145/3485447.3512204">ONBRA: Rigorous Estimation of the Temporal Betweenness Centrality in Temporal Networks.</a></emph> WWW</li>
<li>Diego Santoro, Leonardo Pellegrina, Matteo Comin, Fabio Vandin <emph><a href="https://doi.org/10.1093/bioinformatics/btac180">SPRISS: approximating frequent k-mers by sampling reads, and applications.</a></emph> Bioinform.</li>
<li>Fabio Vandin <emph><a href="https://doi.org/10.1145/3535334">Technical perspective: Evaluating sampled metrics is challenging.</a></emph> Commun. ACM</li>
<li>Johann Gamper, Matteo Ceccarello, Anton Dignös <emph><a href="https://doi.org/10.1007/978-3-031-15740-0_5">What&apos;s New in Temporal Databases?</a></emph> ADBIS</li>
<li>Leonardo Pellegrina, Cyrus Cousins, Fabio Vandin, Matteo Riondato <emph><a href="https://doi.org/10.1145/3532187">MCRapper: Monte-Carlo Rademacher Averages for Poset Families and Approximate Pattern Mining.</a></emph> ACM Trans. Knowl. Discov. Data</li>
<li>Leonardo Pellegrina, Fabio Vandin <emph><a href="https://pubmed.ncbi.nlm.nih.gov/36124798/">Discovering significant evolutionary trajectories in cancer phylogenies</a></emph> Bioinformatics</li>
<li>Martin Aumüller , Matteo Ceccarello <emph><a href="https://doi.org/10.5441/002/edbt.2022.07">Implementing Distributed Similarity Joins using Locality Sensitive Hashing.</a></emph> EDBT</li>
<li>Martin Aumüller , Sariel Har-Peled, Sepideh Mahabadi, Rasmus Pagh, Francesco Silvestri  <emph><a href="https://doi.org/10.1145/3543667">Sampling near neighbors in search for fairness.</a></emph> Commun. ACM</li>
<li>Martin Aumüller , Sariel Har-Peled, Sepideh Mahabadi, Rasmus Pagh, Francesco Silvestri  <emph><a href="https://doi.org/10.1145/3502867">Sampling a Near Neighbor in High Dimensions - Who is the Fairest of Them All?</a></emph> ACM Trans. Database Syst.</li>
<li>Matteo Ceccarello, Johann Gamper <emph><a href="https://www.vldb.org/pvldb/vol15/p3841-ceccarello.pdf">Fast and Scalable Mining of Time Series Motifs with Probabilistic Guarantees.</a></emph> Proc. VLDB Endow.</li>
<li>Paolo Pellizzoni, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.3390/a15020052">k-Center Clustering with Outliers in Sliding Windows.</a></emph> Algorithms</li>
<li>Paolo Pellizzoni, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.1007/s41060-022-00318-z">Adaptive k-center and diameter estimation in sliding windows.</a></emph> Int. J. Data Sci. Anal.</li>
<li>Paolo Sylos Labini, Massimo Bernaschi, Werner Nutt, Francesco Silvestri , Flavio Vella <emph><a href="https://doi.org/10.1109/IA356718.2022.00009">Blocking Sparse Matrices to Leverage Dense-Specific Multiplication.</a></emph> IA3@SC</li></ul>
<h3 class="mb-0">2021</h3><ul><li>Andrea Tonon, Fabio Vandin <emph><a href="https://doi.org/10.1109/ICDM51629.2021.00075">CASPITA: Mining Statistically Significant Paths in Time Series Data from an Unknown Network.</a></emph> ICDM</li>
<li>Davide Buffelli, Fabio Vandin <emph><a href="https://ieeexplore.ieee.org/document/9382331">Attention-Based Deep Learning Framework for Human Activity Recognition With User Adaptation</a></emph> IEEE Sensors</li>
<li>Diego Santoro, Leonardo Pellegrina, Fabio Vandin <emph><a href="https://arxiv.org/abs/2101.07117">SPRISS: Approximating Frequent k-mers by Sampling Reads, and Applications</a></emph> RECOMB</li>
<li>Elia Costa, Francesco Silvestri  <emph><a href="https://doi.org/10.4230/OASIcs.ATMOS.2021.5">On the Bike Spreading Problem.</a></emph> ATMOS</li>
<li>Federico Altieri, Andrea Pietracaprina, Geppino Pucci, Fabio Vandin <emph><a href="https://doi.org/10.1137/1.9781611976700.73">Scalable Distributed Approximation of Internal Measures for Clustering Evaluation.</a></emph> SDM</li>
<li>Ilie Sarpe, Fabio Vandin <emph><a href="https://doi.org/10.1145/3459637.3482459">odeN: Simultaneous Approximation of Multiple Motif Counts in Large Temporal Networks.</a></emph> CIKM</li>
<li>Ilie Sarpe, Fabio Vandin <emph><a href="https://doi.org/10.1137/1.9781611976700.17">PRESTO: Simple and Scalable Sampling Techniques for the Rigorous Approximation of Temporal Motif Counts.</a></emph> SDM</li>
<li>Martin Aumüller , Matteo Ceccarello <emph><a href="https://doi.org/10.1016/j.is.2021.101807">The role of local dimensionality measures in benchmarking nearest neighbor search.</a></emph> Inf. Syst.</li>
<li>Martin Aumüller , Sariel Har-Peled, Sepideh Mahabadi, Rasmus Pagh, Francesco Silvestri  <emph><a href="https://doi.org/10.1145/3471485.3471496">Fair near neighbor search via sampling.</a></emph> SIGMOD Rec.</li>
<li>Matteo Comin, Barbara Di Camillo, Cinzia Pizzi, Fabio Vandin <emph><a href="https://doi.org/10.1093/bib/bbaa121">Comparison of microbiome samples: methods and computational challenges.</a></emph> Briefings Bioinform.</li>
<li>Rezaul Chowdhury, Francesco Silvestri , Flavio Vella <emph><a href="https://doi.org/10.1007/978-3-030-85665-6_22">Algorithm Design for Tensor Units.</a></emph> Euro-Par</li></ul>
<h3 class="mb-0">2020</h3><ul><li>Andrea Tonon, Fabio Vandin <emph><a href="https://doi.org/10.1109/ICDM50108.2020.00064">GRosSo: Mining Statistically Robust Patterns from a Sequence of Datasets.</a></emph> ICDM</li>
<li>Diego Santoro, Andrea Tonon, Fabio Vandin <emph><a href="https://doi.org/10.3390/a13050123">Mining Sequential Patterns with VC-Dimension and Rademacher Complexity.</a></emph> Algorithms</li>
<li>Leonardo Pellegrina, Cinzia Pizzi, Fabio Vandin <emph><a href="https://doi.org/10.1089/cmb.2019.0314">Fast Approximation of Frequent k-Mers and Applications to Metagenomics.</a></emph> J. Comput. Biol.</li>
<li>Leonardo Pellegrina, Cyrus Cousins, Fabio Vandin, Matteo Riondato <emph><a href="https://doi.org/10.1145/3394486.3403267">MCRapper: Monte-Carlo Rademacher Averages for Poset Families and Approximate Pattern Mining.</a></emph> KDD</li>
<li>Leonardo Pellegrina, Fabio Vandin <emph><a href="https://doi.org/10.1007/s10618-020-00687-8">Efficient mining of the most significant patterns with permutation testing.</a></emph> Data Min. Knowl. Discov.</li>
<li>Martin Aumüller , Matteo Ceccarello <emph><a href="https://doi.org/10.1007/978-3-030-60936-8_31">Running Experiments with Confidence and Sanity.</a></emph> SISAP</li>
<li>Martin Aumüller , Rasmus Pagh, Francesco Silvestri  <emph><a href="https://doi.org/10.1145/3375395.3387648">Fair Near Neighbor Search: Independent Range Sampling in High Dimensions.</a></emph> PODS</li>
<li>Matteo Ceccarello, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.1145/3402448">A General Coreset-Based Approach to Diversity Maximization under Matroid Constraints.</a></emph> ACM Trans. Knowl. Discov. Data</li>
<li>Matteo Ceccarello, Andrea Pietracaprina, Geppino Pucci, Eli Upfal <emph><a href="https://doi.org/10.3390/a13090216">Distributed Graph Diameter Approximation.</a></emph> Algorithms</li>
<li>Matteo Riondato, Fabio Vandin <emph><a href="https://doi.org/10.1145/3385653">MiSoSouP: Mining Interesting Subgroups with Sampling and Pseudodimension.</a></emph> ACM Trans. Knowl. Discov. Data</li>
<li>Paolo Pellizzoni, Andrea Pietracaprina, Geppino Pucci <emph><a href="https://doi.org/10.1109/DSAA49011.2020.00032">Dimensionality-adaptive k-center in sliding windows.</a></emph> DSAA</li>
<li>Rezaul Chowdhury, Francesco Silvestri , Flavio Vella <emph><a href="https://doi.org/10.1145/3350755.3400252">A Computational Model for Tensor Core Units.</a></emph> SPAA</li>
<li>Thomas D. Ahle, Francesco Silvestri  <emph><a href="https://doi.org/10.1007/978-3-030-60936-8_6">Similarity Search with Tensor Core Units.</a></emph> SISAP</li>
<li>Yoo-Ah Kim, D. Wojtowicz, R. Sarto Basso, I. Sason, W. Robinson, D. S. Hochbaum, M. Leiserson, R. Sharan, F. Vandin, T. Przytycka. <emph><a href="https://pubmed.ncbi.nlm.nih.gov/32471470/">Network-based approaches elucidate differences within APOBEC and clock-like signatures in breast cancer</a></emph> Genome Medicine</li></ul>
<!-- PUBS -->
            </section>
            <hr class="m-0" />
            <!-- Projects-->
            <section class="resume-section" id="projects">
                <div class="resume-section-content">
                    <h2 class="mb-5">Recent Projects</h2>
                    <ul>
                      <li>[2021-2024] European Union, <strong>Brainteaser: BRinging Artificial INTelligencE home for a better cAre of amyotrophic lateral sclerosis and multiple SclERosis.</strong> Role: participants. <span style="font-style:italic">Total UniPD amount: euro 732.250.</span></li>
                      <li>[2019-2023] Ministero dell’Istruzione, dell’Universit`a e della Ricerca (MIUR), <strong>PRIN AHeAD: efficient Algorithms for HArnessing networked Data.</strong> Role: co-PI. <span style="font-style:italic">Total amount: euro 784,860.</span>
                      <li>[2018-2020] University of Padova, <strong>STARS: Algorithms for Inferential Data Mining.</strong>  Role: PI. <span style="font-style:italic">otal amount: euro 140,000.</span></li>
                      <li>[2017-2019] University of Padova, <strong>Algorithms for Networks Analysis and Bioinformatics Applications.</strong>  Role: PI. <span style="font-style:italic">Total amount: euro 54,000.</span></li>
                      <li>[2017-2020] University of Padova, <strong>From Single-Cell to Multi-Cells Information Systems Analysis.</strong>  Role: co-PI. <span style="font-style:italic">Total amount: euro 220,000.</span></li>
                      <li>[2012-2018]National Science Foundation, <strong>BIGDATA: Mid-Scale: Analytical Approaches to Massive Data Computation with Applications to Genomics.</strong> Role: co-PI <span style="font-style:italic">Total amount: $1,566,685.</span></li>
                    </ul>
            </section>
            <hr class="m-0" />
            <!-- Awards-->
            <section class="resume-section" id="awards">
                <div class="resume-section-content">
                    <h2 class="mb-5">Awards</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Best Paper Award</strong>, EURO-PAR 2023 (29th International European Conference of Parallel and Distributed Computing)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Best Paper Award</strong>, SSDBM 2023 (35th International Conference on Scientific and Statistical Database Management)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Best Paper Award</strong>, ECML-PKDD 2022 (European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases 2022)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Runner-up for Test of Time award</strong>, RECOMB 2022 (Conference on Research in Computational Molecular Biology 2022)
                        </li>

                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Invited article to the special issue of KAIS for the best papers</strong> of IEEE ICDM 2020 (International Conference on Data Mining)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Best Paper Award</strong> of SISAP 2019 (International Conference on Similarity Search and Applications)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Invited article to the special issue of ACM TKDD for the best papers</strong> of ACM KDD 2018 (International Conference on Knowledge Discovery and Data Mining)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Invited article to the special issue of Algorithms for Molecular Biology for the best papers</strong> of WABI 2018 (Workshop on Algorithms for Bioinformatics)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Best Paper Award</strong>, RECOMB 2013 (Conference on Research in Computational Molecular Biology 2013)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Invited article to the special issue of Algorithms for Molecular Biology for the best papers</strong> of WABI 2011 (Workshop on Algorithms for Bioinformatics)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            <strong>Invited article to the special issue of DAMI for the best papers</strong> of ECML-PKDD 2010 (European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases 2010)
                        </li>
                      </ul>
                </div>
            </section>
        </div>
        <!-- Theses-->
        <section class="resume-section" id="theses">
            <div class="resume-section-content">
                <h2 class="mb-5">Master Theses</h2>
                <p>
                Click <a href="assets/AIDA_theses_Dec2022.pdf">HERE</a> to download a presentation of the topics available for Master (i.e., <span style="font-style:italic">Laurea Magistrale</span>) theses in our lab (last update: December 2022)
                </p>
                <p>
                <strong>Important:</strong> we only supervise students in Computer Engineering Master degree or with a strong background in algorithms.
              </p>

        </section>
        <hr class="m-0" />
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
